reversion/revisions.py

```diff
@@ -407,6 +414,19 @@ def register(model):
     return register(model)
 
 
+def _parse_register_arguments(fields, exclude, follow, format, for_concrete_model, ignore_duplicates,
+                              use_natural_foreign_keys):
+    return SimpleNamespace(
+        fields=fields or app_settings.DEFAULT_FIELDS,
+        exclude=exclude or app_settings.DEFAULT_EXCLUDE,
+        follow=tuple(follow or app_settings.DEFAULT_FOLLOW),
+        format=format or app_settings.DEFAULT_FORMAT,
+        for_concrete_model=for_concrete_model or app_settings.DEFAULT_FOR_CONCRETE_MODEL,
+        ignore_duplicates=ignore_duplicates or app_settings.DEFAULT_IGNORE_DUPLICATES,
+        use_natural_foreign_keys=use_natural_foreign_keys or app_settings.DEFAULT_USE_NATURAL_FOREIGN_KEYS,
```

Comment:
This shortcut logic isn't quite right. You want to use the pattern:

```
fields=app_settings.DEFAULT_FIELDS if fields is None else fields 
```

Otherwise, an empty string or tuple will count as "unset"

Reply:
Fixed :+1: